parachain/pallets/broker/src/lib.rs

```diff
@@ -45,7 +47,7 @@ decl_module! {
 		#[weight = 0]
 		pub fn accept(origin, app_id: AppID, message: Message) -> DispatchResult {
 			// we'll check the origin here to ensure it originates from a verifier
-			ensure_root(origin)?;
+			//ensure_root(origin)?;
```

Comment:
Why has this been commented out?

Reply:
Oops, I commented that while testing submission of extrinsics from the relayer. Will push a fix.